utils/tests/test_split_event_config.json

```diff
@@ -0,0 +1,65 @@
+{
+    "config":{
+        "resolver_file_path": "default",
+        "filtered_events_table_name":"test_split_events",
```

Comment:
Are we changing this filename with the rename?

Reply:
Yep!